release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,195 +1,223 @@
+
+* green: issues/PRs where the resolution is known and will be resolved soon
+* yellow: issues/PRs which are in progress but are not as easy and need to be watched
+* red: issues/PRs that are heavily release-impacting, and/or with a likelihood of not making it in time.
+
+In each meeting, it is good to give an overall status of Bug Triage: Green, Yellow or Red, generally based on the amount of issues from each category. Examples are:
+
+- status can be Red if there are multiple release-impacting issues that are in danger of not being fixed in the release.
+- status can be Yellow if there are multiple issues open but they are being worked on, which is the usual state of the world.
+- status can be Green if there are no Red issues and workpaces seem normal.
+
+It is useful to keep Red issues on a separate section and report them during the meetings.
+
+
 ## Code Freeze
 
-Once Code Freeze starts, all issues still in the upcoming release **should** have a priority label of `priority/critical-urgent`.  This means that on the morning before Code Freeze begins you need to go through the open issues (and PRs) which are approved-for-milestone but not marked critical-urgent and poke the owners reminding them that in the next day Code Freeze will mean any issues not marked `priority/critical-urgent` will potentially be moved to the subsequent release. 
+Once Code Freeze kicks in, only PRs targeting the upcoming release (/milestone v1.xx) are able to get merged. This is done to stabilize the release as much as possible, and reduce the overall tracking footprint to focus on the upcoming release.
 
-The criteria for moving issues is a mix of:
+This is enforced via test-infra automation, as a means of "new release coming in a few weeks - we're only merging relevant PRs in".
 
-- How critical is the Issue to be fixed for the currently imminent release?
-- Is the PR expected to be ready for merging soon?
-- How big is the change? Could it cause instability to CI? If so, it either needs to be closely tracked /or/ moved.
+As explained in sections [Code Freeze Preparation](#code-freeze-preparation) and [Filtering](#filtering), it becomes incrementally more difficult to merge new stuff as time progresses. As such, in the days around the start of Code Freeze, owners of issues and PRs should be poked again, reminding them that Code Freeze starts/started at DD/MM.
 
-For issues/PRs that look tricky, you can consolidate with the Release Lead and SIG/PR owners to reach a decision.
+The criteria for moving issues are described in [Filtering](#filtering).
 
-After this, you need to monitor all of these issues to make sure that daily progress is made on them.  Theoretically, issue owners/SIGs are supposed to make a daily status comment in the issue, but this is seldom followed. You also will need to send daily reminders/queries about stuck PRs.
+For issues/PRs that look tricky, you can always consolidate with the Release Lead and SIG/PR owners to reach a decision.
```

Comment:
"coordinate" instead of "consolidate" maybe?

Reply:
clearer yes